dash-renderer/src/persistence.js

```diff
@@ -265,10 +265,18 @@ const noopTransform = {
     apply: (storedValue, _propValue) => storedValue
 };
 
-const getTransform = (element, propName, propPart) =>
-    propPart
-        ? element.persistenceTransforms[propName][propPart]
-        : noopTransform;
+const getTransform = (element, propName, propPart) => {
+    if (element.persistenceTransforms) {
```

Comment:
Since both 
```
    if (element.persistenceTransforms) {
        if (element.persistenceTransforms[propName]) {
```
result in `return noopTransform, this can be simplified to a single `if`

With the newest [conditional chaining syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) this would become:
```
if (element.persistenceTransforms?.[propName]) { ... }
```
can also be be written with the more habitual
```
    if (element.persistenceTransforms && element.persistenceTransforms[propName]) {
```